src/elements/fields/SignatureField.jsx

```diff
@@ -24,13 +25,16 @@ function SignatureField({
         penColor='black'
         canvasProps={{
           id: servar.key,
-          width: element.styles.width,
           height: element.styles.height,
-          style: applyStyles.getTarget('field', true)
+          style: {
```

Comment:
why is width being set here and applyWidth is being called on styles?

Reply:
img 1 - shows the width that gets set with `applyWidth` 
img 2 - shows how the canvas stretches outside the parent when there is no width set directly on the canvas
img 3 - shows how setting the width to 100% makes it occupy the width of its parent. Sorry - I was wrong about setting the width to `${element.styles.width}${element.styles.width_unit}`

<img width="1718" alt="image" src="https://user-images.githubusercontent.com/923485/161333545-ad290fa3-2262-48e8-b4f4-7e5bfdd6975b.png">
<img width="1717" alt="image" src="https://user-images.githubusercontent.com/923485/161333624-3b7bd9aa-cd01-4045-b178-b40909688a60.png">
<img width="1721" alt="image" src="https://user-images.githubusercontent.com/923485/161334231-ba3853a9-8017-4b22-99b6-17e7ac2a1f16.png">

Reply:
oh, sorry I published a new form between images 1 and 3 changing the width to 80%. that's why the canvas doesn't take up the whole column in the 3rd image

this is how it looks at 100%
<img width="872" alt="image" src="https://user-images.githubusercontent.com/923485/161335028-98ae7c5a-5bba-41e4-9114-eb0905022bf4.png">
<img width="1720" alt="image" src="https://user-images.githubusercontent.com/923485/161335059-fdbfd1b3-54cc-4cde-8825-ad2975e6e976.png">